src/modules/auth/components/Form/RegisterForm/RegisterForm.jsx

```diff
@@ -110,11 +114,13 @@ export const RegisterForm = () => {
       <Input
         {...register("confirmPassword")}
         type="password"
-        placeholder="Hasło"
+        placeholder="Powtórz hasło"
         label="Powtórz hasło"
         classes={styles.form__input}
       />
-      <p className={styles.form__error}>{errors.confirmPassword?.message}</p>
+      <span className={styles.form__error}>
```

Comment:
WDYT about making from this error span a component called `ErrorBox`? Thanks to that it will be easy to use and reusable

Reply:
I think it's good idea but IMO it's a bit too late

Reply:
It is just moving code from one place to another one. I don't think it gonna take a big amount of time